lib/Service/CardService.php

```diff
@@ -274,14 +274,17 @@ public function delete($id) {
 	 * @param $description
 	 * @param $order
 	 * @param $duedate
+	 * @param $deletedAt
+	 * @param $archived
+	 * @param $done
 	 * @return \OCP\AppFramework\Db\Entity
 	 * @throws StatusException
 	 * @throws \OCA\Deck\NoPermissionException
 	 * @throws \OCP\AppFramework\Db\DoesNotExistException
 	 * @throws \OCP\AppFramework\Db\MultipleObjectsReturnedException
 	 * @throws BadRequestException
 	 */
-	public function update($id, $title, $stackId, $type, $owner, $description = '', $order = 0, $duedate = null, $deletedAt = null, $archived = null) {
+	public function update($id, $title, $stackId, $type, $owner, $description = '', $order = 0, $duedate = null, $deletedAt = null, $archived = null, $done = null) {
```

Comment:
```suggestion
	public function update($id, $title, $stackId, $type, $owner, $description = '', $order = 0, $duedate = null, $deletedAt = null, $archived = null, ?bool $done = null) {
```

Reply:
We can use type hints these days, so would be nice to at least already add them for new arguments or new methods :)

Reply:
Uh okay I was just following the rest of the codebase. Fixed along with all the other type hits in d24bdc1